modules/aws/centos-gfx/centos-gfx-startup.sh.tmpl

```diff
@@ -172,21 +175,29 @@ install_pcoip_agent() {
         fi
         log "--> PCoIP agent installed successfully."
 
-        log "--> Register PCoIP agent license ..."
-        n=0
-        while true; do
-            /usr/sbin/pcoip-register-host --registration-code="$PCOIP_REGISTRATION_CODE" && break
-            n=$[$n+1]
-
-            if [ $n -ge 10 ]; then
-                log "--> Failed to register PCoIP agent after $n tries."
-                exit 1
-            fi
+        if [[ "${pcoip_registration_code}" ]]
```

Comment:
So this would allow PCoIP reg code to still work for other deployments, so we're not completely moving everything to lls?

Reply:
yes. this module is shared between CLS and LLS, so it needs to work for both.